gatsby-theme-landing-page/src/components/button.js

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import { Link } from "gatsby";
+import * as styles from "./button.module.css";
+
+export default function Button({ href, text, children, variant = "primary" }) {
```

Comment:
Cool - not necessary for this PR, but I'm thinking we pull these "base" components out to their own directory e.g. `src/components/base`, to differentiate them from the Contentful section components. WDYT?

Reply:
I was thinking something similar, but with the top-level components. I'm guessing we'll have some other shared components to pull out, so definitely prefer to wait until we have the basics working, then take an audit of what we have and organize as needed. The more directories we create, the more complex that makes the API for end-users of the theme